ccd-definition/CaseEvent/CareSupervision/Gatekeeping-nonprod.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "addGatekeepingOrder",
+    "Name": "V2: Add the gatekeeping order",
+    "Description": "Add the gatekeeping order",
+    "DisplayOrder": 10,
+    "PreConditionState(s)": "Gatekeeping",
```

Comment:
when porting existing functionality (https://tools.hmcts.net/jira/browse/FPLA-3067) update state to include case management